src/common/sockman.cpp

```diff
@@ -10,6 +10,19 @@
 #include <util/sock.h>
 #include <util/thread.h>
 
+CService GetBindAddress(const Sock& sock)
```

Comment:
Maybe yes, but `sock.{h,cpp}` does not currently include `netaddress.h` and I guess including it may cause circular dependency. It would also increase the scope of this PR. I am leaving it as it is.